validator/src/journal/commit_store_ffi.rs

```diff
@@ -19,13 +19,15 @@ use std::mem;
 use std::os::raw::{c_char, c_void};
 use std::slice;
 
-use protobuf;
 use sawtooth::journal::commit_store::{
     ByHeightDirection, CommitStore, CommitStoreByHeightIterator,
 };
-use sawtooth::{batch::Batch, block::Block, transaction::Transaction};
-use transact::database::error::DatabaseError;
-use transact::database::lmdb::LmdbDatabase;
+use sawtooth::transact::database::error::DatabaseError;
```

Comment:
This is missing pulling in Transaction and Batch from transact.

Reply:
Fixed